main.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+    _ "fmt"
+    "os"
+    "io/ioutil"
+    "bytes"
+    "flag"
+    "bufio"
+)
+
+
+func main() {
+    /* Go won't allow the same arg/flag construction we had before, so all flags instead */
+    project := flag.String("project", "", "Project name")
+    multisite := flag.Bool("multisite", false, "Make it a multisite")
+    flag.Parse()
+    if *project == "" {
+        println("You must set a value for the project argument")
+        os.Exit(1)
+    }
+    if *multisite == true {
+        println("MULTISITE")
+    }
+
+    makedir("bin")
+    makedir("setup")
+    makedir("setup/content")
```

Comment:
I think it would make sense to move the calls to `makedir()` into `creating()`. The function to fetch the directory name from a path is: https://godoc.org/path#Dir

Reply:
That would make sense, but one of the directories (`setup/content`) doesn't actually receive any files, so never gets referenced by `creating()`. So instead, I've tried to simplify this a bit by just setting up an array of required directories, then just calling `makeDirs()` once with that array. I've also used `MkdirAll()` within that function so that subdirectories don't have to be created separately.